Sources/Recycling/ListViews/Notifications/Demo/NotificationsListViewDemoViewHelpers.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+import Foundation
+
+public struct NotificationGroup: NotificationsGroupListViewModel {
+    public var attributedTitle: NSAttributedString
+    public var timeAgo: String
+    public var footerAction: String
+    public var notifications: [NotificationsListViewModel]
+}
+
+/// A model confirming to the NotificationsListViewModel protocol for showcasing NotificationsListViewCell in playground.
+public struct Notification: NotificationsListViewModel {
+    public var imagePath: String?
+    public var imageSize: CGSize
+    public var detail: String
+    public var title: String
+    public var price: String
+
+    public var accessibilityLabel: String {
+        var message = detail
+        message += ". " + title
+        message += ". Pris: kroner " + price
```

Comment:
Currency in VoiceOver needs the units after the price without whitespace so that it won't get interpreted as a phone number (reading individual digits). I would suggest: `message += ". Pris: \(price)kroner"`.

Reply:
You need to make a blog post about this! Some good tips you got there.

Reply:
That's basically all my knowledge summarized in two sentences. 😅